usim/_primitives/timing.py

```diff
@@ -420,46 +420,53 @@ def __repr__(self):
         yield loop.time
 
 
-def each(
-        *, delay: float = None, interval: float = None
-) -> AsyncIterable[float]:
+def interval(value: float) -> AsyncIterable[float]:
     """
-    Iterate through time by either ``delay`` or ``interval``
+    Iterate through time by ``interval``
 
-    :param delay: on each step, pause for ``delay``
-    :param interval: on each step, pause until ``interval`` since the last step
-    :raises TypeError: if both ``delay`` and ``interval`` are provided
+    :param value: on each step, pause until ``interval`` since the last step
 
     Asynchronous iteration pauses and provides the current time at each step.
 
     .. code:: python3
 
         print('It was', time.now)  # 0
-        async for now in each(delay=10):
-            print('It is', now)  # 10, 20, 30, ...
+        async for now in interval(value=10):
+            await (time + 1)
+            print(now, time.now)  # (10, 11), (20, 21), (30, 31), ...
 
     The first pause occurs *before* entering the loop body.
 
-    Setting `delay`` causes iteration to always *pause for* the same time,
-    even if the current activity is :term:`suspended <Suspension>`
-    in the loop body.
-    Setting ``interval`` causes iteration to *resume at* regular times,
+    Using interval causes iteration to *resume at* regular times,
     even if the current activity is :term:`suspended <Suspension>`
     in the loop body - the pause is shortened if necessary.
 
-    .. code:: python3
+    .. seealso:: :py:func:`~.delay` if you want to always *pause for* the same time
+    """
+    return each_interval(value)
```

Comment:
You can pull in the ``each_interval`` function directly. It is not public and not used elsewhere.

Reply:
Applied this.